### DISCUSSION TITLE
第4节课作业(4班)

### DISCUSSION THREAD

**Discussion by FlamingoPg** [InternLM/Tutorial#314] [作业]
请4班同学们把第四节课作业交到这里哦

**Comment by zhjunqin** [comment]
zhjunqin: https://github.com/zhjunqin/InternLM-tutorial/tree/master/Fourth

**Comment by onlyzer0** [comment]
onlyzer0:https://blog.csdn.net/qq_42907812/article/details/135574702?spm=1001.2014.3001.5502

**Comment by helonggood** [comment]
【XTuner 大模型单卡低成本自我认知微调】
http://t.csdnimg.cn/DzoGd

**Comment by phbst** [comment]
https://thankful-archeology-738.notion.site/74a1795867fc4437ba8cb7de2f99388a?pvs=4
